### PR TITLE
Test cure teleport

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

File: src/Goerli-DssSpell.t.sol [hunk 1]
```diff
@@ -565,4 +560,31 @@ contract DssSpellTest is GoerliDssSpellTestBase {
             8 days
         );
     }
+
+    // NOTE: Only executable by forge
```
Comment (hexonaut) [inline src/Goerli-DssSpell.t.sol:564]: This can actually be executed in dapptools just fine.

**Review by hexonaut — APPROVED** [review]
Good to merge after comment is removed.

**Comment by gbalabasquer** [comment]
I'm not sure I really like much how these functions with `cage` flag are built and executed. It cost me a bit to understand what was happening. I would think that it is just better to get that extracted out to the test itself and maybe the internal in the base contract wouldn't even be needed at all. But I'm fine if you guys prefer this way. After fixing Sam's comment, should be good to merge.

File: src/Goerli-DssSpell.t.sol (hunk 1 above)
Comment (talbaneth) [reply src/Goerli-DssSpell.t.sol]: Done.

**Pull request merged**